cache_images/podman_tooling.sh

```diff
@@ -14,6 +14,23 @@ REPO_DIRPATH=$(realpath "$SCRIPT_DIRPATH/../")
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+if [[ "$OS_RELEASE_ID" == "ubuntu" ]]; then
+    if [[ -n "$(type -P bats)" ]]; then
+        die "Bats _MUST_ not be installed on ubuntu until fixed: https://bugs.launchpad.net/ubuntu/+source/bats/+bug/1882542"
+    fi
+    req_env_vars UBUNTU_BATS_VERSION
+    # shellcheck disable=SC2154
+    echo "Installing bats $UBUNTU_BATS_VERSION to /usr/local"
+    buildDir=$(mktemp -d)
+    lilto git clone -b "$UBUNTU_BATS_VERSION" \
+        https://github.com/bats-core/bats-core $buildDir
```

Comment:
I think I'd be a lot happier with:
```bash
  wget https://github.com/bats-core/bats-core/archive/v${UBUNTU_BATS_VERSION}.tar.gz
  tar -C ... xvf    <--- goes into bats-core-x.y.z
```
Do you have a strong reason for doing it via git?